dist/templates/ovnkube-node.yaml.j2

```diff
@@ -32,6 +32,7 @@ spec:
       # Requires fairly broad permissions - ability to read all services and network functions as well
       # as all pods.
       serviceAccountName: ovn
+      {{ "hostPID: true" if ovn_unprivileged_mode=="off" }}
       hostNetwork: true
       hostPID: true
```

Comment:
You're still setting `hostPID` unconditionally

Reply:
actually no, see if statement... it will not set it if feature is disabled

Reply:
@dyusupov Please see L37 :)

Reply:
good catch! :-) typo